strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/OrientDbUserService.java

```diff
@@ -0,0 +1,205 @@
+package org.carlspring.strongbox.users.service.impl;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+import javax.inject.Qualifier;
+
+import org.apache.commons.lang3.StringUtils;
+import org.carlspring.strongbox.data.CacheName;
+import org.carlspring.strongbox.data.service.CommonCrudService;
+import org.carlspring.strongbox.domain.UserEntry;
+import org.carlspring.strongbox.users.domain.UserData;
+import org.carlspring.strongbox.users.domain.Users;
+import org.carlspring.strongbox.users.dto.User;
+import org.carlspring.strongbox.users.security.SecurityTokenProvider;
+import org.carlspring.strongbox.users.service.UserEntryService;
+import org.carlspring.strongbox.users.service.impl.OrientDbUserService.OrientDb;
+import org.carlspring.strongbox.users.userdetails.StrongboxUserDetails;
+import org.jose4j.lang.JoseException;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Component;
+
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import com.orientechnologies.orient.core.sql.OCommandSQL;
+import com.orientechnologies.orient.core.sql.query.OSQLSynchQuery;
+
+/**
+ * @author sbespalov
+ */
+@Component
+@OrientDb
+public class OrientDbUserService extends CommonCrudService<UserEntry> implements UserEntryService
+{
+
+    @Inject
+    private SecurityTokenProvider tokenProvider;
+    
+    @Override
+    public void deleteByUsername(String username)
+    {
+        Map<String, String> params = new HashMap<>();
+        params.put("username", username);
+
+        String sQuery = String.format("delete from %s where username = :username", UserEntry.class.getSimpleName());
+
+        OCommandSQL oQuery = new OCommandSQL(sQuery);
+        getDelegate().command(oQuery).execute(params);
+    }
+
+    @Override
+    public UserEntry findByUserName(String username)
```

Comment:
Cosmetic: This should be called `findByUsername`.

Reply:
fixed

Reply:
Thanks! :)